packages/tools/kadena-cli/src/utils/simulationOptions.ts

```diff
@@ -0,0 +1,69 @@
+import { Option } from 'commander';
+import { z } from 'zod';
+import { createOption } from './createOption.js';
+
+import type { ChainId } from '@kadena/types';
+import { simulate as simulationPrompts } from '../prompts/index.js';
+export const simulationOptions = {
+  //Simulation
+  simulationNumberOfAccounts: createOption({
+    key: 'simulationNumberOfAccounts' as const,
+    prompt: simulationPrompts.numberOfAccountsPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-na, --simulation-number-of-accounts <simulationNumberOfAccounts>',
```

Comment:
@alber70g there was some confusion about this
but this should be
-n or only --simulation-number-of-accounts and removal of the -na 

correct | lets get this confusion out :)

Reply:
So since this is a specifically for devnet simulate, should we move it to the `simulate` directory?

Reply:
Moved to simulation specific directory; updated the flags to single characters